src/app/(marketing)/create-profile/_components/StartProfile.tsx

```diff
@@ -9,119 +11,211 @@ import {
-                  <FormControl>
-                    <Input placeholder="" required {...field} />
-                  </FormControl>
-                  <FormErrorMessage />
-                </FormItem>
+      <FormProvider {...form}>
+        <form onSubmit={form.handleSubmit(onSubmit)} className="flex flex-col gap-3">
+          <FormField
+            control={form.control}
+            name="profileImageUrl"
+            render={({ field }) => (
+              <Button
+                variant="unstyled"
+                type="button"
+                disabled={method === undefined}
+                className={cn(
+                  "mb-8 size-36 self-center rounded-full border-2 border-dotted",
+                  { "border-solid": field.value }
+                )}
+                {...getRootProps()}
+              >
+                <Avatar className="size-36 bg-card-secondary">
+                  {field.value && (
+                    <AvatarImage
+                      src={field.value}
+                      width={144}
+                      height={144}
+                      alt="profile avatar"
+                    />
+                  )}
+                  <AvatarFallback className="bg-card-secondary">
+                    {avatarFallBackIcon}
+                  </AvatarFallback>
+                </Avatar>
+              </Button>
+            )}
+          />
+
+          {method === undefined ? (
+            <>
+              {farcasterProfile ? (
+                <Button
+                  className="ml-0 w-full gap-2 bg-farcaster hover:bg-farcaster-hover active:bg-farcaster-active"
+                  size="md"
+                  onClick={() => setMethod(CreateMethod.FillByFarcaster)}
+                >
+                  <FarcasterImage />
+                  Fill using Farcaster
+                </Button>
+              ) : (
+                <ConnectFarcasterButton className="ml-0 w-full gap-2" size="md">
+                  <FarcasterImage />
+                  Connect farcaster
+                </ConnectFarcasterButton>
               )}
-            />
-            <Button
-              className="w-full"
-              type="submit"
-              colorScheme="success"
-              onClick={() => setStartMethod(undefined)}
-              disabled={!form.formState.isValid}
-            >
-              Start my profile
-            </Button>
-          </form>
-        </FormProvider>
-      ) : (
-        <>
-          <ConnectFarcasterButton
-            className="ml-0 w-full gap-2"
-            size="md"
-            onClick={() => setStartMethod("farcaster")}
-          >
-            <FarcasterImage />
-            Connect farcaster
-          </ConnectFarcasterButton>
 
-          <Button variant="ghost">
-            I don't have a Farcaster profile
-            <ArrowRight weight="bold" />
-          </Button>
-        </>
-      )}
+              <Button
+                variant="ghost"
```

Comment:
This could be a "link" variant

Reply:
There is no such variant yet. How should it look like? @BrickheadJohnny

Reply:
It should look like an anchor component actually